### ISSUE TITLE
Taiko reports "exists is not a function" on a valid element

### ISSUE THREAD

**Issue by ajoecker** [getgauge/taiko#27]
### Expected behavior
the verification should pass and only fail if the element can not be found

### Actual behavior
Taiko reports `link(...).exists is not a function`

### Steps to reproduce
1. openBrowser()
2. goto('http://google.de')
3. link('Gmail').exists()

### Versions
Gauge: 0.9.7 (Commit Hash: bc9784c)
taiko: 0.0.9
node v8.10.0
```
Paste `node` and `taiko` versions here.
```

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 21c9289

**Comment by sguptatw** [comment]
**Verified with version**
```
commit 215a0767e673443e8f91ddaa0a56e984bad09a94
```
